File: URI_Python_UFRR/1010.py
linha1 = input().split(" ")
linha2 = input().split(" ")

cp, qp, vp = linha1
cn, qn, vn = linha1
total = ( int(qp)*float(vp) ) + ( int(qn) * float(vn) )
print('VALOR A PAGAR: R$ {0:.2f}'.format(total))

File: URI_C_UFRR/1010.c
#include <stdio.h>
 
int main() {
 
	int n,q,n1,q1;
	float p,p1,total;
	scanf("%i %i %f",&n,&q,&p);
	scanf("%i %i %f",&n1,&q1,&p1);
	total = (q * p) + (q1 * p1);
	printf("VALOR A PAGAR: R$ %.2f\n",total);
	 
    return 0;
}
